packages/tests/src/pytest_plugins/custom_logging/plugin_logging.py

```diff
@@ -317,9 +324,12 @@ def pytest_configure(config: pytest.Config) -> None:
 
     worker_id = os.getenv("PYTEST_XDIST_WORKER", "main")
     log_filename = f"{log_stem}-{worker_id}.log"
-    log_path = Path("logs")
-    log_path.mkdir(exist_ok=True)
-    log_file_path = log_path / log_filename
+
+    # Use --log-to if provided, otherwise default to ./logs
+    log_dir = getattr(config.option, "eest_log_dir", None)
+    base_logs_dir = Path(log_dir) if log_dir else Path("logs")
```

Comment:
How would this interact with `--eest_log_dir ""`, as in "log to the current directory"? Would:

```suggestion
    base_logs_dir = Path(log_dir) if log_dir is not None else Path("logs")
```

Be safer?

Reply:
You are right, using `is not None` is safer here. That way an explicitly empty string `(--eest_log_dir "")` will correctly resolve to the current directory instead of defaulting to `./logs`. I will go ahead and update the code accordingly.